scrapers/koji_scraper.py

```diff
@@ -0,0 +1,154 @@
+# SPDX-License-Identifier: GPL-3.0+
+from scrapers.base import BaseScraper
+from purview.models.koji import KojiBuild, KojiTask
+from purview.models.user import User
+from datetime import datetime
+
+import logging
+
+log = logging.getLogger(__name__)
+
+
+class KojiScraper(BaseScraper):
+
+    def run(self, since=None):
+        log.info('Connected to TEIID, fetching initial load for koji')
+        # Initialize a start date from which all builds must be fetched
+        # Currently hardcoding this to fetch builds from the past two years
+        if since is None:
```

Comment:
`since` will be user input, which will likely be a string. Let's add this to the base class. I'll submit a PR for this shortly.

Reply:
I filed #9